app/services/sendgrid_service.rb

```diff
@@ -94,6 +103,20 @@ def sendgrid_email_formation(email, name = nil)
 
   private
 
+  def send_email_for_order(order, template_name)
+    return unless user.email.present?
+    begin
+      @add_bcc = true
+      substitution_hash.merge!(user.email_properties)
+      substitution_hash.merge!(order.email_properties)
+      @mail.from = sendgrid_email_formation(ENV["APPOINTMENT_FROM_EMAIL"], I18n.t("email_from_name"))
+      @mail.template_id = template_id(template_name)
+      send_email
+    rescue => e
```

Comment:
Avoid rescuing without specifying an error class.

Reply:
This code was written in orders, I just moved it. I don't know what the error class would be